.flowconfig

```diff
@@ -5,6 +5,8 @@
 [options]
 module.system=node
 log.file=./artifacts/flow.log
+suppress_comment= \\(.\\|\n\\)*\\$FLOW_FIXME
+suppress_comment= \\(.\\|\n\\)*\\$FLOW_IGNORE 
```

Comment:
This `suppress_comment` options in the `".flowconfig"` are useful to ignore (and associate an helpful comment to) flow errors that are raised due to some of the tricks used in the tests or due to flow limitations.

I've defined `// $FLOW_IGNORE: <comment>` to ignore errors that are related to type checking errors that are used on purpose in the testing suite, and `// $FLOW_FIXME: <comment>` to ignore errors that we want to fix in followups (the latter is currently unused).